src/datagen/main/kotlin/mekanism/patchouli/MekanismMainBookProvider.kt

```diff
@@ -0,0 +1,410 @@
+                                    "$(li)It can be placed in the ${link(ELECTROLYTIC_SEPARATOR, "Electrolytic Separator's")} left output slot (where Hydrogen is outputted) after placing water in the machine's input slot" +
+                                    "$(li)It can be placed in a ${link(GuideEntry.TANKS_GAS, "Chemical Tank's")} output slot.$(br2)" +
+                                    "While worn, the Jetpack displays the Hydrogen remaining and the mode active in the bottom left corner."
+                        }
+                        text {
+                            title = "Operation Modes"
+                            text = "The Jetpack has three modes to choose from, which can be toggled by pressing the ${MekanismKeyHandler.chestModeSwitchKey()} key." +
+                                    "$(li)$(bold)Regular$() (default): Press $(k:jump) to increase your height and release to fall. Note that you will take fall damage unless you carefully lower yourself to the ground." +
+                                    "$(li)$(bold)Hover$(): Constant flight, without the need to level yourself like you do with Regular mode. Press $(k:jump) to increase altitude and press $(k:sneak) to decrease. Note that this mode constantly consumes Hydrogen, but at a reduced rate as compared to Regular mode." +
+                                    "$(li)$(bold)Disabled$(): The Jetpack is disabled."
+                        }
+                        text {
+                            title = "Tips"
+                            text = "\$(li)The Jetpack cannot be worn with chestplate armor, since it uses the same slot, consider upgrading it to the ${link(ARMORED_JETPACK, "Armored Jetpack")} if you want protection." +
+                                    "\$(li)The Jetpack emits fire particles; however, it will not set anything on fire."+
+                                    "\$(li)If you want to maintain your altitude, choose Hover mode. If you want to ascend/descend rapidly, use Regular mode. If you want to conserve fuel while trekking across hills, mountains, consider Disabled mode."+
+                                    "\$(li)The Jetpack can be paired with the Free Runners to protect against fall damage."
+                        }
+                    }
+                    ATOMIC_DISASSEMBLER("The Atomic Disassembler is Mekanism's an all-in-one tool, essentially the ultimate, electronic version of the Paxel (working at any mining level). Also functions as a Hoe & Scoop (Forestry)$(p)The Atomic Disassembler has multiple modes that can be cycled with $(k:sneak) + right click.") {
+                        text {
+                            title = "Normal Mode"
+                            text = "Base speed setting, single block.$(li)Roughly equivalent to Efficiency II.$(li)Right click Dirt to till a 3x3 area to Farmland$(li)Right click grass to make a 3x3 Grass Path, right click again to till"
```

Comment:
This isn't actually true anymore the right click to till, make grass paths, and strip logs got moved to the meka tool's farming module.

Reply:
Ah, good spotting. I've mostly been copying from the wiki